### DISCUSSION TITLE
Server to Server Events (or Broadcasting)

### DISCUSSION THREAD

**Discussion by dellow** [laravel/framework#42181] [General]
Hi All,

Am I right in thinking it's not currently possible to publish an event (or broadcast) from one Laravel app and consume it in an entirely separate app?

I've used broadcasting a lot, but it seems built for server to client communication not server to server. Ideally though I'd like to use the EventServiceProvider and publish events to a service like AWS SNS or Pusher.

I've tried a lot of Googling to get an answer for this but it's not really widely covered. I would have thought in the world of microservices this might be extremely useful. 

As a side note, I'm **not** building a bunch microservices here, just two apps that occasionally need to talk to each other.

Thanks.

**Comment by Sophist-UK** [comment]
AFAIK broadcasting is more server to client, and events is more internal to an instance of Laravel.

Event communications between different Laravel instances in different webservers is effectively RPC, but I expect that it might be possible using Broadcasting.

**Comment by Sophist-UK** [comment]
This discussion was mentioned in https://github.com/laravel/framework/discussions/43311.
